### DISCUSSION TITLE
Gesamtproduktion/Tagesproduktion in OpenDTU zurücksetzen?

### DISCUSSION THREAD

**Discussion by H83v** [tbnobody/OpenDTU#1425] [General]
Hi, 

gibt es eine Möglichkeit die Ertragsdaten, die im Display meiner OpenDTU (in Verbindung mit HM HMS 800) zurück zusetzen.
Z.B. wenn ich nach 12 Monaten, den Gesamtertrag fürs neue Jahr auf 0 zurück setzen möchte?

Grüße Hannes

P.S. ich möchte allen Beteiligten hier mal ein riesen Lob aussprechen. Bin recht neu im Thema OpenDTU und Github allgemein.
Mit meinen kleinen IT Kenntnissen kommt man Dank dem Support hier oft weiter :-)

**Comment by Juergen2453** [comment]
Der "Gesamtertrag Insgesamt" kommt aus dem Inverter und kann nicht zurückgesetzt werden.
Du kannst aber in OpenDTU einen Offset setzen damit die Anzeige wieder von 0 anfängt.

![wechselrichter](https://github.com/tbnobody/OpenDTU/assets/116970574/f60ddbd9-7cc8-4a3b-a501-66808bd11f39)

**Reply by H83v** [reply]
@Juergen2453 
Helf mir noch kurz weiter. Aktuell habe ich bei dem "String" Reiter in den Wechselrichtereinstellungen nur den Namen meiner beiden Strings geändert. "Max. Leistung String 1+2" steht auf "0", wie auch der "Ertragsversatz String 1+2".
Meine Module machen laut Hersteller max. 375Wp. Was bewirkt es, wenn ich das unter "Max. Leistung String 1+2" eintrage? 
Welche Vorteile hab ich dadurch?

Zum "Ertragsversatz String 1+2": Wie gehe ich im Detail vor? Wird der Gesamtertrag vom Wechselrichter oder von den einzelnen Strings damit zurück gesetzt? Passiert das dann da täglich oder kann ich den Wert auch einfach 1x im Jahr zurücksetzen?
Sorry viele Fragen, aber klar ist mir es im Moment noch nicht.

VG Hannes

**Comment by Juergen2453** [comment]
Siehe Gesamtertrag, So sieht es bei mir aus

![g3](https://github.com/tbnobody/OpenDTU/assets/116970574/605af3fb-7e72-4259-a63a-341aa1c051a5)

dann den Eintrag pro String

![g2](https://github.com/tbnobody/OpenDTU/assets/116970574/2b701093-cd8e-4381-825c-70946a902a57)

Das Ergebnis ist dann Gesamtertrag=0. Ab dem Zeitpunkt wo du das einträgst fängt er an hochzuzählen (als wenn du einen neuen Inverter gekauft hast) Wenn du wieder 0 einträgst sind die originalwerte wieder da. 

![g1](https://github.com/tbnobody/OpenDTU/assets/116970574/4d46d700-ec83-4664-8f08-7f6f1cf0d6c2)

**Reply by H83v** [reply]
Das macht es mir als Laie verständlicher. Super, danke dir!

**Comment by Juergen2453** [comment]
Zitat: "Was bewirkt es, wenn ich das unter "Max. Leistung String 1+2" eintrage"
Wenn dort Werte drin stehen werden noch andere Berechnungen durchgeführt z.B. Einstrahlung

**Comment by canone2** [comment]
Super ,Danke! Hat auch mir geholfen nach einem WR Wechsel.

**Comment by canone2** [comment]
Hallo thnobody,
Danke für den Magic Moment!Es hat wunderbar geklappt mit dem Gesamtertrag!
Eine kurze Frage:Da ich noch auf openDTU generic bin und auf ESP32 gehen wollte würde mich interessieren ob ich irgentwas beachten muss?  Gruß

**Reply by tbnobody** [reply]
Da die Frage öfter kommt habe ich es in die Dokumentation aufgenommen. <https://tbnobody.github.io/OpenDTU-docs/firmware/howto/migrate_generic/>
